.github/workflows/make_release.yml

```diff
@@ -97,11 +96,161 @@ jobs:
+            | base64 -d | gunzip | patch -p1
+
+          CREDENTIAL_FILE_NAME=$(echo ${GOOGLE_APPLICATION_CREDENTIALS} | awk -F/ '{print $NF}')
+          cp scripts/release-binary.sh scripts/release-binary.sh.tmp
+          echo gcloud auth activate-service-account --key-file="/work/${CREDENTIAL_FILE_NAME}" > scripts/release-binary.sh
+          cat scripts/release-binary.sh.tmp >> scripts/release-binary.sh
+          sed -i 's/release release\-symbol debug/release release\-symbol/' scripts/release-binary.sh
+          sed -i -n '/Build and publish Wasm plugins/q;p' scripts/release-binary.sh
+          cat scripts/release-binary.sh
+
+      - name: Build and push
+        if: ${{ steps.check_already_built.outputs.should_build == '1' }}
+        env:
+          BUILD_WITH_CONTAINER: 1
+          IMG: gcr.io/tetrate-istio-arm/build-tools-proxy:release-${{ steps.get_minor_ver.outputs.RELEASE_VERSION }}-tid
+        run: make push_release
+
+      - name: Put CentOS binary
+        # Copy the binary built on Ubuntu to CentOS path - this binar cannot run on CentOS/RHEL 7, but fine with CentOS 8.
+        # The point is that we cannot build FIPS binary directly on CentOS 7 due to the constraints described in the BoringCrypto certification.
+        run: |
+          SHA=$(git rev-parse --verify HEAD)
+          gsutil cp ${RELEASE_GCS_PATH}/envoy-alpha-${SHA}.tar.gz ${RELEASE_GCS_PATH}/envoy-centos-alpha-${SHA}.tar.gz
+
+      - name: Put Wasm binary
+        # Copy the built Wasm binary in the upstream to the bucket - Wasm is nothing to do with FIPS stuff.
+        env:
+          UPSTREAM_GCSP_PATH: gs://istio-build/proxy
+        run: |
+          SHA=$(git rev-parse --verify HEAD)
+          gsutil cp "${UPSTREAM_GCSP_PATH}/metadata_exchange-${SHA}*" ${RELEASE_GCS_PATH}/
+          gsutil cp "${UPSTREAM_GCSP_PATH}/stats-${SHA}*" ${RELEASE_GCS_PATH}/
```

Comment:
from 1.18 istio move to use native stats filter

Reply:
added a condition in workflow.